user-manager/src/main/java/com/qpay/usermanager/mapper/AuthenticationUserMapper.java

```diff
@@ -0,0 +1,59 @@
+package com.qpay.usermanager.mapper;
+
+import com.qpay.libs.models.UserType;
+import com.qpay.usermanager.model.dto.authuser.AuthenticationUserCreation;
+import com.qpay.usermanager.model.dto.authuser.AuthenticationUserModification;
+import com.qpay.usermanager.model.dto.customer.CustomerModification;
+import com.qpay.usermanager.model.dto.merchant.MerchantCreation;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AuthenticationUserMapper {
+
+    public AuthenticationUserCreation mapCustomerCreation(final CustomerModification customerModification) {
```

Comment:
Customer creation but you pass modification.
Your class is ugly . Check it yourself